src/__tests__/linter.test.ts

```diff
@@ -152,16 +150,11 @@ describe('linter', () => {
 
   test('should not report anything for disabled rules', async () => {
     spectral.addFunctions(oas3Functions());
-    const oas3Rules = await readRulesFromRulesets('spectral:oas3');
-    spectral.addRules(mergeRulesets(
-      { rules: oas3Rules },
-      {
-        rules: {
-          'valid-example': 'off',
-          'model-description': -1,
-        },
-      },
-    ).rules as RuleCollection);
+    const { rules: oas3Rules } = await readRuleset('spectral:oas3');
+    spectral.addRules(mergeRules(oas3Rules, {
```

Comment:
I'm adding setRules in #325 if you just wanna grab that? might be a bit easier. I use it to update the rules after skipRules() in the CLI.

Reply:
Added.
I also have `setRuleset` in one of my other PRs, that we probably could recommend using.